src/test/regress/expected/pg14.out

```diff
@@ -7,14 +7,102 @@ SELECT substring(:'server_version', '\d+')::int > 13 AS server_version_above_thi
 \endif
 create schema pg14;
 set search_path to pg14;
+SET citus.next_shard_id TO 980000;
+SET citus.shard_count TO 2;
+-- test the new vacuum option, process_toast
+CREATE TABLE t1 (a int);
+SELECT create_distributed_table('t1','a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SET citus.log_remote_commands TO ON;
+VACUUM (FULL) t1;
+NOTICE:  issuing VACUUM (FULL) pg14.t1_980000
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+NOTICE:  issuing VACUUM (FULL) pg14.t1_980000
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+NOTICE:  issuing VACUUM (FULL) pg14.t1_980001
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+NOTICE:  issuing VACUUM (FULL) pg14.t1_980001
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+VACUUM (FULL, PROCESS_TOAST) t1;
+NOTICE:  issuing VACUUM (FULL,PROCESS_TOAST) pg14.t1_980000
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+NOTICE:  issuing VACUUM (FULL,PROCESS_TOAST) pg14.t1_980000
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+NOTICE:  issuing VACUUM (FULL,PROCESS_TOAST) pg14.t1_980001
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+NOTICE:  issuing VACUUM (FULL,PROCESS_TOAST) pg14.t1_980001
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+VACUUM (FULL, PROCESS_TOAST true) t1;
+NOTICE:  issuing VACUUM (FULL,PROCESS_TOAST) pg14.t1_980000
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+NOTICE:  issuing VACUUM (FULL,PROCESS_TOAST) pg14.t1_980000
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+NOTICE:  issuing VACUUM (FULL,PROCESS_TOAST) pg14.t1_980001
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+NOTICE:  issuing VACUUM (FULL,PROCESS_TOAST) pg14.t1_980001
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+VACUUM (FULL, PROCESS_TOAST false) t1;
+ERROR:  PROCESS_TOAST required with VACUUM FULL
```

Comment:
shall we run it without `FULL` so that we show that we can send `FALSE` option properly?